.github/workflows/cronet.yml

```diff
@@ -19,48 +21,45 @@ env:
   PUB_ENVIRONMENT: bot.github
 
 jobs:
-  analyze:
-    name: Lint and static analysis
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: pkgs/cronet_http
+  verify:
+    name: Format & Analyze & Test
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        package: ['cronet_http', 'cronet_http_embedded']
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: '17'
       - uses: subosito/flutter-action@v2
         with:
-          # TODO: Change to 'stable' when a release version of flutter
-          # pins version 1.21.1 or later of 'package:test'
-          channel: 'master'
+          channel: 'stable'
+      - name: Make cronet_http_embedded copy
+        if: ${{ matrix.package == 'cronet_http_embedded' }}
+        run: |
+          cp -r pkgs/cronet_http pkgs/cronet_http_embedded
+          cd pkgs/cronet_http_embedded
+          flutter pub get && dart tool/prepare_for_embedded.dart
       - id: install
         name: Install dependencies
+        working-directory: 'pkgs/${{ matrix.package }}'
         run: flutter pub get
       - name: Check formatting
+        working-directory: 'pkgs/${{ matrix.package }}'
         run: dart format --output=none --set-exit-if-changed .
         if: always() && steps.install.outcome == 'success'
       - name: Analyze code
+        working-directory: 'pkgs/${{ matrix.package }}'
         run: flutter analyze --fatal-infos
         if: always() && steps.install.outcome == 'success'
-
-  test:
-    # Test package:cupertino_http use flutter integration tests.
-    needs: analyze
-    name: "Build and test"
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-java@v4
-        with:
-          distribution: 'zulu'
-          java-version: '17'
-      - uses: subosito/flutter-action@v2
-        with:
-          channel: 'stable'
       - name: Run tests
         uses: reactivecircus/android-emulator-runner@v2
+        if: always() && steps.install.outcome == 'success'
         with:
           api-level: 28
           target: playstore
```

Comment:
Shouldn't this be false for cronet_embedded?

Reply:
Not sure why we have `false` here. Can you explain more?

Reply:
Sorry, I meant that you probably do **not** want to use the playstore when testing cronet_http_embedded because it is designed to work without the playstore. I think that the value should be "default" for cronet_http_embedded. See https://github.com/ReactiveCircus/android-emulator-runner?tab=readme-ov-file#configurations